backend/tests/data_layer/database/db_connection/sqlite/test_sqlite_db_connection.py

```
@@ -0,0 +1,49 @@
+    with get_session(engine) as session:
+        assert session.is_active
+        assert session.connection
+        assert session.bind
+
+        # Check if the tables are empty
+        try:
+            for model_class in model_classes.values():
+                assert not session.exec(select(model_class)).all()
+
+        except Exception as e:
+            raise AssertionError(f"Failed to interact with the database: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)